bats/fb-katello-client-global-registration.bats

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bats
+# vim: ft=sh:sw=2:et
+
+set -o pipefail
+
+load os_helper
+load foreman_helper
+load fixtures/content
+
+@test "remove subscription manager" {
+  if [ -e "/etc/rhsm/ca/candlepin-local.pem" ]; then
+    rpm -e `rpm -qf /etc/rhsm/ca/candlepin-local.pem`
+  fi
+
+  cleanSubscriptionManager
+  tPackageRemove subscription-manager
```

Comment:
Is that so that the GR command can install it again?

Doesn't this break pulp3 on EL8 where this might kill python3-rhsm (or whatever the right pkg name is)?

Reply:
I hope not. I did not see any breaks with my testing.

Reply:
If you ran it on EL8 and it did not make pulp-certguard go belly up, we're good.

Reply:
Looks all good:

```
==========================================================================================================================================================================================================================================================================================
 Package                                                                              Architecture                                                Version                                                              Repository                                                    Size
==========================================================================================================================================================================================================================================================================================
Removing:
 subscription-manager                                                                 x86_64                                                      1.28.13-3.el8_4                                                      @baseos                                                      4.3 M
Removing unused dependencies:
 dnf-plugin-subscription-manager                                                      x86_64                                                      1.28.13-3.el8_4                                                      @baseos                                                       87 k
 python3-dmidecode                                                                    x86_64                                                      3.12.2-15.el8                                                        @baseos                                                      308 k
 python3-ethtool                                                                      x86_64                                                      0.14-3.el8                                                           @baseos                                                       92 k
 python3-inotify                                                                      noarch                                                      0.9.6-13.el8                                                         @baseos                                                      243 k
 python3-librepo                                                                      x86_64                                                      1.12.0-3.el8                                                         @baseos                                                      172 k
 usermode                                                                             x86_64                                                      1.113-1.el8                                                          @baseos                                                      837 k

Transaction Summary
==========================================================================================================================================================================================================================================================================================
Remove  7 Packages
```